src/targets/symbolCollector.ts

```diff
@@ -103,6 +103,12 @@ export class SymbolCollector extends BaseTarget {
         '--server-endpoint',
         this.symbolCollectorConfig.serverEndpoint,
       ]);
+
+      if (cmdOutput) {
+        this.logger.info(cmdOutput.toString());
+      } else {
+        this.logger.info(`The command didn't have any output.`);
```

Comment:
Thinking about this more, cmdOutput returns either a `Buffer` or `undefined`. `undefined` is only returned in dry run mode, otherwise if the command has no output it'll return an empty array (`[]`).

Therefore we shouldn't log anything out in the else condition. We should check instead for an empty array to say that the command didn't have any output.